app/seaf-cli

```diff
@@ -232,7 +232,10 @@ def get_token(url, username, password, conf_dir):
         'client_version': client_version,
         'platform_version': platform_version,
     }
-    token_json = urlopen("%s/api2/auth-token/" % url, data=data)
+    headers = {
```

Comment:
I think it should be changed to 

```
if (tfa != None) {
   ...
}
```

Otherwise the X-SEAFILE-OTP is always set. It is better to avoid adding an unnecessary header.